torch/testing/_internal/faulty_rpc_agent/rpc.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
```

Comment:
this file should be called `backend_registry.py` to stay consistent with `rpc/backend_registry.py`?

Reply:
Sounds good - I kept the naming conventions similar to Thrift RPC Agent, but I can change it since this is OSS.